combat/warriors.cpp

```diff
@@ -1049,47 +1046,55 @@ int cmdHarmTouch(Player* player, cmd* cmnd) {
 
 
 
-    creature->print("%M hits you with fists shrouded in darkness!\n", player);
-    broadcast(player->getSock(),  creature->getSock(), creature->getRoomParent(), "%M hits %N with fists shrouded in darkness!", player, creature);
+    *creature << player << " touches you with " << player->hisHer() << " malevolent hand!\n";
```

Comment:
this probably needs setf(CAP); (or whatever it is) try it with a possessed mob on live vs after this change.